### PR TITLE
wm_error updates

### PR DESCRIPTION
**Pull request by sezero**
wm_error updates. Also see bug #41

### CODE REVIEW

**Comment by sezero** [comment]
PING ?

**Comment by psi29a** [comment]
No need for ping, I've just been juggling work, family and house renovations. Testing this branch out is on my todo list. :)

**Comment by psi29a** [comment]
I ran through it on linux, gcc5 and clang with valgrind... i tried to run some unhappy tests as well and things look as expected.

**Comment by psi29a** [comment]
How far does this take us in being able to close/resolve https://github.com/Mindwerks/wildmidi/issues/41 ?
It probably isn't a bad idea to use gtest for unit testing. ;)

File: include/wm_error.h [hunk 1]
```diff
@@ -50,14 +50,20 @@ enum {
 extern char * _WM_Global_ErrorS;
 extern int _WM_Global_ErrorI;
 
-extern void _WM_GLOBAL_ERROR(const char * func, const char * file, unsigned int lne, int wmerno, const char * wmfor, int error);
+extern void _WM_GLOBAL_ERROR(const char *func, unsigned int lne, int wmerno, const char * wmfor, int error);
 
+/* sets the global error string to a custom msg */
```
Comment (psi29a) [inline include/wm_error.h]: Thank you very much for adding documentation, even if one-liners.

**Pull request merged**